tests/waku_archive/test_waku_archive.nim

```diff
@@ -58,22 +50,6 @@ suite "Waku Archive - message handling":
     check:
       (waitFor driver.getMessagesCount()).tryGet() == 2
 
-  test "it should archive a message with no sender timestamp":
-    ## Setup
-    let driver = newSqliteArchiveDriver()
-    let archive = newWakuArchive(driver)
-
-    ## Given
-    let invalidSenderTime = 0
-    let message = fakeWakuMessage(ts = invalidSenderTime)
-
-    ## When
-    waitFor archive.handleMessage(DefaultPubSubTopic, message)
-
-    ## Then
-    check:
-      (waitFor driver.getMessagesCount()).tryGet() == 1
-
```

Comment:
Instead of deleting the test case, wouldn't it be better to adapt it to the new behavior when time is not set?